components/tools.ts

```diff
@@ -112,10 +120,27 @@ async function exportDebugInfo(): Promise<void> {
         flags: getAllFlags(),
     }
 
-    await writeFile("DEBUG_PROFILE.json", JSON.stringify(data, undefined, 4))
+    const debugJson = JSON.stringify(data, undefined, 4)
+
+    const zipFile = ppath.resolve(ppath.cwd(), "DEBUG_PROFILE.zip")
+
+    // we'll start by creating an empty zip file
+    await writeFile(zipFile, makeEmptyArchive())
+
+    const zip = new ZipFS(zipFile, { create: true })
+
+    await zip.writeFilePromise(zip.resolve("meta.json" as Filename), debugJson)
+
+    await copyIntoZip(zip, "logs")
+    await copyIntoZip(zip, "userdata")
+    await copyIntoZip(zip, "contractSessions")
+    await copyIntoZip(zip, "contracts")
```

Comment:
I don't think `contracts` is really needed?

Reply:
In case of a contract specific problem? Not sure.